metrics.nim

```diff
@@ -69,7 +69,7 @@ const CONTENT_TYPE* = "text/plain; version=0.0.4; charset=utf-8"
 #########
 
 when defined(metrics):
-  proc printError(msg: string) =
+  proc printError*(msg: string) =
```

Comment:
This procedure was made public because its used in `xxx_httpserver.nim` implementation.

Reply:
Can we avoid exporting these internal procedures with a "metrics/common.nim", or we get surprise exports anyway?